website/docs/usefullpaths.md

```diff
@@ -1,12 +1,20 @@
-usefullpaths - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+Turn on/off full paths usage in diagnostics
 
 ```lua
-usefullpaths (value)
+usefullpaths "switch"
```

Comment:
Most of the On/Off toggle-able values are `("value")` in documentation, so that should probably match.